src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,51 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_NUMBER];
+        values = (V[]) new Object[MAX_NUMBER];
+    }
+
     @Override
     public void put(K key, V value) {
+        if (size == MAX_NUMBER) {
+            throw new RuntimeException("Can't add key: "
+                        + key
+                        + "\nvalue: "
+                        + value
+                        + "Storage is full");
+        }
+        for (int i = 0; i < size; i++) {
+            if (Objects.equals(keys[i], key)) {
```

Comment:
Also, from checklist: **Don’t use class Objects**.
```suggestion
            if (...call here your own method...) {
```